fingerpaintview/src/main/java/tech/picnic/fingerpaintview/FingerPaintImageView.kt

```diff
@@ -164,14 +188,54 @@ class FingerPaintImageView @JvmOverloads constructor(context: Context,
 
     override fun onDraw(canvas: Canvas?) {
         super.onDraw(canvas)
-        paths.forEach { canvas?.drawPath(it.first, it.second) }
+        mBitmap = Bitmap.createBitmap(widthBitmap,
```

Comment:
Compiler is throwing the following warning: `Avoid object allocations during draw/layout operations...`. This warning could be solved by creating the `bitmap` and `canvas` before calling `onDraw(...)`.